ui/app/src/components/PublishingStatusDialog/PublishingStatusDialogBody.tsx

```diff
@@ -93,9 +93,9 @@ function PublishingStatusDialogBody(props: PublishingStatusDialogBodyProps) {
                 <Skeleton />
               ) : (
                 <>
-                  {message && <div>{message}</div>}
-                  {lockOwner && <div>{formatMessage(publishingStatusTileMessages.lockOwner, { lockOwner })}</div>}
-                  {lockTTL && <div>{formatMessage(publishingStatusTileMessages.lockTTL, { lockTTL })}</div>}
+                  {message}
+                  {lockOwner && formatMessage(publishingStatusTileMessages.lockOwner, { lockOwner })}
+                  {lockTTL && formatMessage(publishingStatusTileMessages.lockTTL, { lockTTL })}
```

Comment:
?

Reply:
Fixing a issue related with a <div> being child from <p>

Reply:
This breaks the UI. Can't just do this (test it).
What's the issue?